users/forms.py

```diff
@@ -27,3 +31,28 @@ class Meta:
         model = CustomUser
         fields = "__all__"
         widgets = {"country": CountrySelectWidget()}
+
+
+class CustomUserSignUpForm(UserCreationForm):
+    email = forms.EmailField(max_length=constants.EMAIL_MAX_LENGTH, required=True, label="id_email")
+    password1 = forms.CharField(help_text="", label="id_password1", widget=forms.PasswordInput())
+    password2 = forms.CharField(help_text="", label="id_password2", widget=forms.PasswordInput())
+
+    class Meta:
+        model = CustomUser
+        fields = ("email", "password1", "password2")
+
+    def save(self, commit: bool = True) -> None:
+        user = super(CustomUserSignUpForm, self).save(commit=False)
+        user.email = self.cleaned_data["email"]
```

Comment:
This is not needed, email should be already set on user.

Reply:
Done

Reply:
Why is it still there then? :)